iac-extensions/docker/src/test/java/org/sonar/iac/docker/tree/impl/ExposeTreeImplTest.java

```diff
@@ -57,8 +58,8 @@ void exposeInstructionWithSimpleValue() {
 
```

Comment:
Maybe it will be good to add some `notMatches()` to method `matchingSimple()` (GitHub doesn't allow me to put a comment in line 48).